packages/blockchain-wallet-v4-frontend/src/scenes/Dex/Onboarding/components/StepCard/StepCard.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+import { Button, Flex, Padding, Pager } from '@blockchain-com/constellation'
+import styled from 'styled-components'
+
+import { media } from 'services/styles'
+
+const Card = styled.div`
```

Comment:
In that case Flex for some reason doesn't work well together with styled because of custom width and background. Leaving it as div for now. I have some thoughts about Flex implementation in general, we can discuss later as well.